index.html

```diff
@@ -55,6 +55,7 @@
       repo: '',
       loadSidebar: true,
       subMaxLevel: 3,
+      routerMode: 'history',
```

Comment:
Please note that change `routerMode` will breaks browser bookmarks and browser history.